src/main.rs

```diff
@@ -186,6 +195,38 @@ fn _main() -> PaziResult {
         }
     }
 
+    if let Some(_) = flags.subcommand_matches("edit") {
+        let mut fclone = frecency.clone();
```

Comment:
I couldn't find an easy way to avoid this clone; feel free to try if you'd like.

I'm not too worried about performance in the `pazi edit` path, so I didn't try too hard.